octavia-cli/tests/unit/test_apply/test_diff_helpers.py

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+from unittest.mock import mock_open, patch
+
+import pytest
+from octavia_cli.apply import diff_helpers
+
+
+def test_secret_mask():
+    assert diff_helpers.SECRET_MASK == "**********"
```

Comment:
This isn't a huge deal, but it might actually be more detrimental to have this test than it would be to not have it. 

All this test is doing is checking the value of a constant in another file, so it is not actually testing out any logic. So the test is not helpful to have, and it just means that if we ever change the SECRET_MASK to something else, then we have to change it in two places now